### ISSUE TITLE
Verify eth fw version

### ISSUE THREAD

**Issue by aliuTT** [tenstorrent/tt-metal#2939]
For WH, we need to call `verify_eth_fw` to make sure we're not running on incompatible FW. Also need to figure out where to put SW version. Maybe umd?

**Comment by abhullar-tt** [comment]
@aliuTT are we doing this now?

**Comment by aliuTT** [comment]
Nope, we can define a minimum version number for eth FW and have that be checked in ttmetal runtime.

**Comment by prajaramanTT** [comment]
@abhullar-tt @aliuTT Can we consider this ticket done ? If yes, can you please close this ? Thanks

**Comment by aliuTT** [comment]
No, this is not done, but also not urgent, hence P3 priority.
